### ISSUE TITLE
Suggestion: Improved error messages for bad fastqs that cause problems

### ISSUE THREAD

**Issue by morganx** [tseemann/nullarbor#265]
Hi Torsten,
Thanks for making Nullarbor - we really like it. A small suggestion - we spent a while this week trying to figure out why Nullarbor was quietly dying during assembly without generating any helpful messages in the log files, and eventually figured out it was because one of the hundreds of isolates had a corrupt fastq input. Adding a couple of lines of code to check whether read1 and read2 have the same number of lines and generate a helpful warning if they don't may save users many hours of troubleshooting.

**Comment by tseemann** [comment]
Yes some more sanity checking could be added, but i did assume basic QC was done at some point before Nullarbor, but yes things can still go wrong.

`egrep 'warn|error` nullarbor.log`  is a good starting point but I will add more stuff.

Thanks for the feedback!
